.gitignore

```diff
@@ -13,4 +13,6 @@ Localize/out/
 /packages/playground/resources/i18n/*
 /packages/react/resources/i18n/*
 /packages/service/resources/i18n/*
-/util/bux/resources/i18n/json
+/util/bux/resources/i18n
```

Comment:
A couple options that would avoid needing to write a file that needs to be `.gitignore`'d specially for this test:

- Pass in an output path instead of `platform` to `createEnglishTranslations()` and write to a temp directory
or
- Refactor createEnglishTranslations() into 2 functions: one that returns a JSON string (or object) so that you can assert the translated data is correct, and one that writes that data to disk. The latter function could be tested separately by stubbing out the fs module to avoid a 'real' write, or just have it write to a temp file.